cross/build-android-rootfs.sh

```diff
@@ -136,7 +137,11 @@ dpkg -x $__Android_Cross_Dir/deb/libunwind_1.2.20170304_$__AndroidArch.deb $__An
 
 cp -R $__Android_Cross_Dir/tmp/$__AndroidArch/data/data/com.termux/files/usr/* $__ToolchainDir/sysroot/usr/
 
+# Generate platform file for build.sh script to assign to __DistroRid
+# TODO: replace with logic that generates RID based on $__ApiLevel and $__BuildArch
+echo "Generating platform file..."
 
+echo "RID=android.21-arm64" > $__ToolchainDir/sysroot/android_platform
```

Comment:
@chcosta The TODO was addressed in previous commit by adding logic to `build-android-rootfs.sh`; I accidentally left the comment. Will remove comment and echo commands.